app/components/Navbar.tsx

```diff
@@ -15,9 +15,7 @@ import { useMemo } from 'react';
 
 import { NetworkButton } from './NetworkButton';
 import { trackEvent } from 'fathom-client';
-import BetsIcon from '@/app/components/ui/icons/BetsIcon';
-import AIStars from '@/public/ai-stars.svg';
-import Image from 'next/image';
```

Comment:
We removed these 2 but we're not removing the `<Link />` component that implements the button and image depending on these 2 imports.

Do we still need these imports?
Should we remove the implementation at line 58?